### ISSUE TITLE
error log

### ISSUE THREAD

**Issue by tozcutekstil12** [PrestaShop/PrestaShop#29097]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

![php error log](https://user-images.githubusercontent.com/109485216/179424817-1f1206a7-9732-46c1-96e8-edbec46110ea.png)

### Expected behavior

hello can you help me how to fix error log error

### Steps to reproduce

Error log

### PrestaShop version(s) where the bug happened

1.7.8.5

### PHP version(s) where the bug happened

7.4

**Comment by hibatallahAouadni** [comment]
Hello @tozcutekstil12 


Thanks for your issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must provide and respect all blocks in the template to submit a complete bug report**.

Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it.

Thank you

**Comment by tozcutekstil12** [comment]
hi, I don't have any information about why the error log occurs.we have provided a transition from version 1.6 to version 1.7.8.5. And we changed the theme.we used the php version 7.4, which prestashop recommends for version 1.7.8.5. The errors I shared in the error log file occurred geldi.Ve we do not know the reason for these errors, we are waiting for your help in this regard. Our Website www.istanbultoptanicgiyim.com
